File: src/Services/StockDelivery/StockDelivery.API/Queries/V1/Dtos/ActiveDeliveryDto.cs
using System;
using StockDelivery.API.Domain;

namespace StockDelivery.API.Queries.V1.Dtos
{
    public class ActiveDeliveryDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public short BooksCount { get; set; }
        public short ItemsCount { get; set; }
        public bool IsAllDeliveryItemsScanned { get; set; }
        public bool IsAnyDeliveryItemsScanned { get; set; }
        
        public DeliveryStatus DeliveryStatus { get; set; }
        public DateTime ModificationDate { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: src/Web/OrganisationSPA/src/app/features/delivery/models/active-delivery-dto.ts
export class ActiveDelivery {
    id: number
    name: string
    booksCount: number
    itemsCount: number
    isAllDeliveryItemsScanned: boolean
    isAnyDeliveryItemsScanned: boolean

    deliveryStatus: number
    modificationDate: Date | string
    creationDate: Date | string
}
